include/gatekeeper_gk.h

```diff
@@ -114,13 +134,16 @@ struct gk_config {
 	unsigned int       mailbox_burst_size;
 
 	/* Log level for GK block. */
-	uint32_t          log_level;
+	uint32_t           log_level;
```

Comment:
Why are we editing this and other fields in `struct gk_config` that are not being affected by this patch? Do we need to add the space here? If needed, add another patch to this pull request to edit these fields.